workspaces/quay/plugins/quay-common/.eslintrc.js

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright 2024 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+module.exports = require('@backstage/cli/config/eslint-factory')(__dirname);
```

Comment:
Community plugins uses this:

```suggestion
module.exports = require('../../.eslintrc.cjs');
```

Reply:
I think you meant that workspaces/quay/.eslintrc.js should have this change?
Making this change in workspaces/quay/plugins/quay-common/.eslintrc.js gives me an error. 
I also see that in your tekton migration PR you have left it as is: https://github.com/backstage/community-plugins/pull/1914/files#diff-9cf98ab6e4d58818286649976f98df6292b3cfe8c50ab50adad08ddfa6f0ba6c